CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - On Windows, change the default window size (1024x768) to match the default on other desktop platforms (800x600).
 - On Windows, fix bug causing mouse capture to not be released.
 - On Windows, fix fullscreen not preserving minimized/maximized state.
+- macOS: Wake up run loop immediately when request redraw is received.
```

Comment:
```suggestion
- On macOS, wake up the run loop immediately when a redraw is requested.
```
Nit: Shouldn't this added at the top rather than at the bottom?

Reply:
@maroider fixed, thanks!